project/dexter-core/src/java/com/samsung/sec/dexter/core/config/DexterConfigFile.java

```diff
@@ -453,6 +467,9 @@ private String getExistingModuleFullPathWithSourceDirList(){
 		
 		for(String srcDir : this.sourceDirList){
 			File baseFile = new File(srcDir);
+			if(baseFile == null){
```

Comment:
baseFile cannot be null.
srdDir also can't be null, which throw NullPointerException, so you don't need to check.
Read this API doc : https://docs.oracle.com/javase/7/docs/api/java/io/File.html#File(java.lang.String)